code/WorkInProgress/actuallyKeelinsStuff.dm

```diff
@@ -94,6 +94,20 @@ Returns:
 			I.filters += filter(type="layer", render_source = (islist(render_source_cap) ? pick(render_source_cap) : render_source_cap), transform=M2)
 		I.transform = UNLINT(matrix().Turn(-angle).Translate((dist),0).Turn(angle))
 		result.lineImage = I
+	else if(mode == LINEMODE_STRETCH_NO_CLIP)
```

Comment:
could this be turned into a switch statement instead? a suggestion

Reply:
could be. im not really fussed though as the end result wont be something like a nice, compact C# switch, it's still going to be huge